content/tutorial-escrow.md

```diff
@@ -564,19 +568,24 @@ Request:
 _Websocket_
 
 ```json
-{% include 'code_samples/escrow/websocket/account_objects-request-receiver.json' %}
+{% include 'code_samples/escrow/websocket/account_objects-request-sender.json' %}
 ```
 
 <!-- MULTICODE_BLOCK_END -->
 
+
+The response will look something like the following example. Note that the response includes all pending escrow objects with `rfztBskAVszuS3s5Kq7zDS74QtHrw893fm` as the receiver or sender address, where the receiver address is the `Destination` value and the sender address is the `Account` value.
+
+In this case, the first and third escrow objects have `rfztBskAVszuS3s5Kq7zDS74QtHrw893fm` as their receiver address (`Destination`) value and are what we are looking for.
```

Comment:
Another case where I'd prefer you replace "and are what we are looking for" (or make it a stand-alone sentence just for ease of reading).

Reply:
Addressed, my friend... Thank you!